### PR TITLE
Add New Service: Pushover

### PR DESCRIPTION
**Pull request by thebradleysanders**
### Pull Request check-list

To ensure your Pull Request can be accepted as fast as possible, make sure to review and check all of these items:

- [ ] If your changes affects code, did your write the tests?
- [x] Are tests passing? (`npm test` on both front/server)
- [x] Is the linter passing? (`npm run eslint` on both front/server)
- [x] Did you run prettier? (`npm run prettier` on both front/server)
- [x] If you are adding a new features/services, did you run integration comparator? (`npm run compare-translations` on front)
- [ ] If your changes modify the API (REST or Node.js), did you modify the API documentation? (Documentation is based on comments in code)
- [ ] If you are adding a new features/services which needs explanation, did you modify the user documentation? See [the GitHub repo](https://github.com/GladysAssistant/v4-website) and the [website](https://gladysassistant.com).
- [ ] Did you add fake requests data for the demo mode (`front/src/config/demo.json`) so that the demo website is working without a backend? (if needed) See [https://demo.gladysassistant.com](https://demo.gladysassistant.com).

NOTE: these things are not required to open a PR and can be done afterwards / while the PR is open.

### Description of change

Add pushover support as a method of communication, I also changed the UI for scenes->Send Message to support service selection.

### CODE REVIEW

**Comment by thebradleysanders** [comment]
This is my first attempt to contribute to the code base, please let me know what I can do to improve.
I still need to run tests.
Thank you.

**Comment by VonOx** [comment]
@thebradleysanders I've translated in french , i will test your integration ASAP

```json    
"pushover": {
      "title": "Pushover",
      "description": "Recevez des notifications avec Pushover",
      "introduction": "Pour connecter Gladys à Pushover, vous devez créer un compte et configurer une clé API.",
      "note": "Ce lien est unique et spécifique à votre compte, ne le communiquez à personne.",
      "configurationError": "Impossible de configurer le compte. Êtes vous certain que le jeton est valide?",
      "apiKey": "Clé API",
      "userKey": "Clé Utilisateur",
      "saveButton": "Enregistrer"
    }
```

Don't forget to run linter and prettier ( test fail because of that )

Thank you for your contribution btw

**Comment by thebradleysanders** [comment]
@VonOx, Could you point me in the right direction for the Codecov tests? I cannot figure out how to make this work. Thank you.

**Comment by VonOx** [comment]
Hi @thebradleysanders, codecov is working.

test-server is failing in circleci because of:
```
prettier --check '**/*.js' '**/*.json'

Checking formatting...
[error] test/lib/message/message.sendToUser.test.js: SyntaxError: Unexpected token (66:86)
[error]   64 |       telegram_user_id: 'one-id',
[error]   65 |     });
[error] > 66 |     const promise = messageHandler.sendToUser('user-not-found', 'service-not-found'. 'coucou');
[error]      |                                                                                      ^
[error]   67 |     return assertChai.isRejected(promise);
[error]   68 |   });
[error]   69 | });
```

You need to run prettier before commiting your change and you have a typo ( comma missing )
```js
const promise = messageHandler.sendToUser('user-not-found', 'service-not-found', 'coucou');
```

File: server/test/lib/message/message.sendToUser.test.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ describe('message.sendToUser', () => {
       id: '0cd30aef-9c4e-4a23-88e3-3547971296e5',
       telegram_user_id: 'one-id',
     });
-    const promise = messageHandler.sendToUser('user-not-found', 'coucou');
+    const promise = messageHandler.sendToUser('user-not-found', 'service-not-found'. 'coucou');
```
Comment (VonOx) [inline server/test/lib/message/message.sendToUser.test.js]: comma missing

**Comment by Pierre-Gilles** [comment]
Hi ! Any news on this ? :)

The code coverage is low on the following files: 

Screenshot 2021-01-25 at 11 26 57

You can click on the message to see on Codecov where the coverage is low. :)

**Comment by thebradleysanders** [comment]
Sorry no changes I seem to struggle with the code coverage.

**Comment by Pierre-Gilles** [comment]
No worries. Let us know if you have specific questions on the code coverage :)

**Pull request closed without merging**